docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Testing the server-side rendering"
+tags: 
+    - performance
+---
+
+JavaScript handles data fetching, templating, and routing on the client-side rather than on the server-side. However, the amount of JavaScript shipped to the browser tends to increase as the complexity of an application grows. As a result, JavaScript code competes for processing power and delays the rendering task while striving to be processed before the browser finishes rendering the page content.
+
+To ensure your server-side rendering (SSR) HTML is functional and free of bugs, it is important to check how your web page looks and behaves when JavaScript is disabled.
+
+## Step by step
+
+1. Check the documentation of your web browser and search how to disable JavaScript.
+
+    - If you are using Google Chrome, refer to Google's documentation on [Disabling JavaScript.](https://developers.google.com/web/tools/chrome-devtools/javascript/disable).
+
+2. Access your website and observe if its core functionalities are working as expected. Your store will not function exactly as planned but check below which are the expected and undesired behaviors when disabling JavaScript:
```

Comment:
```suggestion
2. Access your store website and observe if its core functionalities are working as expected. Your store will not function exactly as planned but check below which are the expected and undesired behaviors when disabling JavaScript:
```

Reply:
> Access your store website and observe if its core functionalities are working as expected.

Talvez deixar mais claro o que seriam essas core functionalities, algo: "Access your store website and observe if its core functionalities are working as expected, such as product searching, shopping cart and checkout process."

Reply:
@Mariana-Caetano vc acha que vale colocar um link pra página [VTEX Core Services](https://developers.vtex.com/docs/guides/getting-started#vtex-core-services)?